test/blackbox-tests/test-cases/pkg/ocamlformat-dev-tool.t

```diff
@@ -471,14 +471,14 @@ An important cleaning here
   $ rm -fr dune.lock
   $ dune clean
 
-It fails during the build because of missing Ocamlformat module.
+It fails during the build because of missing OCamlFormat module.
   $ DUNE_CONFIG__LOCK_DEV_TOOL=enabled dune fmt
   Solution for dev-tools.locks/ocamlformat:
   - ocamlformat.0.26.4
   File "dev-tools.locks/ocamlformat/ocamlformat.pkg", line 4, characters 6-10:
   4 |  (run dune build -p %{pkg-self:name} @install))
             ^^^^
-  Error: Logs for package ocamlformat
+  Error: Logs for package `ocamlformat`
```

Comment:
This is not a comment, it is printed by dune, so doing this will break the test.